modules/odc-rolling-back-helm-release.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-working-with-helm-charts-using-developer-perspective.adoc
+
+[id="odc-rolling-back-helm-release_{context}"]
+= Rolling back a Helm release
+
+If a release fails to go as per plans, you can rollback the Helm release to a previous version.
+
+.Procedure
+To rollback a release using the *Helm* view:
+
+. In the *Developer* perspective, navigate to the *Helm* view to see the *Helm Releases* in the namespace.
+. Click the *Options* menu {kebab} adjoining the listed release, and select *Rollback* to see the *Rollback Helm Release* page.
+. Select the *Revision* you want to rollback to and click *Rollback*.
```

Comment:
suggestion - In the *Rollback Helm Release* page, select..

Reply:
Done